main.py

```diff
@@ -61,6 +61,8 @@ def check_events(self):
                     self.score = [0, 0]
                 if self.help_pos.collidepoint(pg.mouse.get_pos()):
                     self.show_help = not self.show_help
+        self.turn_text = pg.font.Font(None, 64).render(
```

Comment:
This should be placed where other texts are being initialized rather than in `check_events`